miden-tx/src/host/mod.rs

```diff
@@ -276,6 +292,50 @@ impl<A: AdviceProvider> TransactionHost<A> {
         Ok(())
     }
 
+    // ADVICE INJECTOR HANDLERS
+    // --------------------------------------------------------------------------------------------
+
+    /// Returns a signature as a response to the `SigToStack` injector.
+    ///
+    /// This signature is created during transaction execution and stored for use as advice map
+    /// inputs in the proving host. If not already present in the advice map, it is produced by
+    /// the host's authenticator.
```

Comment:
minor nit: in the last sentence, I would change "it is produced by the host's authenticator" to "it is requested from the host's authenticator".